src/System.Diagnostics.Process/tests/ProcessStreamReadTests.cs

```diff
@@ -13,6 +13,7 @@ namespace System.Diagnostics.Tests
     public class ProcessStreamReadTests : ProcessTestBase
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "https://github.com/dotnet/corefx/issues/22174")]
```

Comment:
Looks like all tests are being skipped on this file, which means that it's better to just add a SkipOnTargetFramework to the class, instead of each test individually.